@xen-orchestra/proxy-cli/src/index.js

```diff
@@ -85,13 +86,33 @@ ${pkg.name} v${pkg.version}`
     request.port = port
   }
 
-  const lines = pumpify.obj(await hrp.post(request), split2())
+  const response = await hrp.post(request)
+
+  const { stdout } = process
+  const { httpVersion, statusCode, statusMessage } = response
+  stdout.write(
+    inspect(`HTTP/${httpVersion} ${statusCode} ${statusMessage}`, {
+      colors: true,
+      depth: null,
+    })
+  )
+  stdout.write('\n')
+
+  const responseType = contentType.parse(response).type
+  stdout.write(inspect(`Content-Type: ${responseType}`, { colors: true, depth: null }))
+  stdout.write('\n\n')
+
+  if (responseType === 'application/octet-stream') {
+    stdout.write('Binary data')
```

Comment:
Shouldn't we handle this case?

Reply:
We don't know what kind of data is and what we should do with this data.

Reply:
We should allow the user to save this data.

Resolved in another comment.